elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1335,12 +1334,12 @@ protected void setValueChecked(String fieldName, Object newValue) {
         ChangeSpec spec = new ChangeSpec(this, fieldName, existingValue, newValue);
         boolean isNewlyCreated = requestScope.getNewPersistentResources().contains(this);
 
-        Class<? extends Annotation> annotationClass = (isNewlyCreated)
-                ? OnCreatePreSecurity.class  : OnUpdatePreSecurity.class;
+        if (!isNewlyCreated) {
+            runTriggers(OnUpdatePreSecurity.class, CLASS_NO_FIELD);
+            runTriggers(OnUpdatePreSecurity.class, fieldName, Optional.of(spec));
+            checkFieldAwareDeferPermissions(UpdatePermission.class, fieldName, newValue, existingValue);
```

Comment:
This looks correct but is actually not right.  It gets remapped to create permission in ActivePermissionExectutor.